website/source/docs/configuration/index.html.md

```diff
@@ -105,6 +105,9 @@ to specify where the configuration is.
   the standard Vault API address will automatically redirect there. This can also
   be provided via the environment variable `VAULT_UI`.
 
+- `pid_file` `(string: "")` - Path to the file in which Vault server PID should
```

Comment:
I'd say "Path to the file in which the Vault server's Process ID (PID) should be stored."

Just in case people think PID is something Vault-y. You never know...